docs/excel/build-your-first-excel-add-in.md

```diff
@@ -251,52 +256,62 @@ You’ll start by creating a web app using HTML and JQuery. Then you'll create a
+#### Deploy the web app and update the manifest
+
+1-  Deploy the QuarterlySalesReportWeb folder to the web server of your choice.
+
+2-  Edit the attribute value within the **SourceLocation** element of the XML manifest file (**QuarterlySalesReportManifest.xml**) to specify the location of the **Home.html** file on the web server.
+
+#### Test the add-in
+
+To test your add-in on Windows, complete the following steps to publish the XML manifest file to a network file share and [sideload the add-in](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) in Excel. 
 
-1-  Create a folder on a network share (for example, \\\MyShare\\QuarterlySalesReport) and copy all the files to that folder.
+> **Note:** For details about testing your add-in in either Office Online or on iPad or Mac, see [Sideload Office Add-ins in Office Online for testing](../testing/sideload-office-add-ins-for-testing.md) or [Sideload Office Add-ins on iPad and Mac for testing](../testing/sideload-an-office-add-in-on-ipad-and-mac.htm.md).
 
-2-  Edit the **SourceLocation** element of the manifest file so that it points to the share location for the .html page from step 1.
+1-  Create a folder on a network share (for example, \\\MyShare\\MyManifests) and copy the XML manifest file to that folder.
 
-3-  Copy the manifest (QuarterlySalesReportManifest.xml) to a network share (for example, \\\MyShare\\MyManifests).
+2-  Now, let's add the share location that contains the manifest as a trusted app catalog in Excel. Launch Excel and open a blank workbook.
 
-4-  Now, let's add the share location that contains the manifest as a trusted app catalog in Excel. Launch Excel and open a blank spreadsheet.
+3-  Choose the **File** tab, and then choose **Options**.
 
-5-  Choose the **File** tab, and then choose **Options**.
+4-  Choose **Trust Center**, and then choose the **Trust Center Settings** button.
 
-6-  Choose **Trust Center**, and then choose the **Trust Center Settings** button.
+5-  Choose **Trusted Add-in Catalogs**.
 
-7-  Choose **Trusted Add-in Catalogs**.
+6-  In the **Catalog Url** box, enter the path to the network share you created in step 1, and then choose **Add Catalog**. Select the **Show in Menu** check box, and then choose **OK**. A message appears to inform you that your settings will be applied the next time you start Office.
 
-8-  In the **Catalog Url** box, enter the path to the network share you created in step 3, and then choose **Add Catalog**. Select the **Show in Menu** check box, and then choose **OK**. A message appears to inform you that your settings will be applied the next time you start Office.
+7-  Restart Excel and open a blank workbook.
 
-9-  Now, let's test and run the add-in. On the **Insert tab** in Excel 2016, choose **My Add-ins**.
+8-  Now, let's test and run the add-in. On the **Insert tab** in Excel 2016, choose **My Add-ins**.
```

Comment:
Fixed.  (I actually removed "To test and run the add-in," since the entire section is about testing and running the add-in -- not necessarily this specific step.)